src/templates/license-manager.defaults.template

```diff
@@ -15,3 +15,4 @@ LM2_AGENT_AUTH0_DOMAIN={{ auth0_domain }}
 LM2_AGENT_AUTH0_AUDIENCE={{ auth0_audience }}
 LM2_AGENT_AUTH0_CLIENT_ID={{ auth0_client_id }}
 LM2_AGENT_AUTH0_CLIENT_SECRET={{ auth0_client_secret }}
+LM2_AGENT_CACHE_DIR=/var/cache/license-manager
```

Comment:
Why is this hard coded?
Isn't the idea to use a env var so we can use a different path when running in other environments?